src/openrct2/Context.cpp

```diff
@@ -1,4 +1,4 @@
-#pragma region Copyright (c) 2014-2017 OpenRCT2 Developers
```

Comment:
Taking out this space is unnecessary. Just modify the end year.

Reply:
See my edit ;-)

Reply:
This is minor, but while you've reintroduced the aforementioned space, you've also introduced spaces around the hyphen. Please remove those — we'd like to keep the copyright line uniform across files.

Reply:
Dumb me of course I would have to remove that issue from clang-format as well. Will fix it with the next revision.